src/my_react/render.ts

```diff
@@ -0,0 +1,85 @@
+const addAttribute = ($elem: HTMLElement, props: JSX.Props, prop: string) => {
+  /* props의 종류에 따라 필요한 처리, 예외 케이스 더 확인되면 추가 필요 */
+  switch (prop) {
+    case 'styles': {
+      Object.keys(props[prop]).forEach((key) => {
+        ($elem.style as any)[key] = props[prop][key];
+      });
```

Comment:
```suggestion
    case 'styles': {
      Object.keys(props.style).forEach((key) => {
        ($elem.style as any)[key] = props[prop][key];
      });
```
여기는 `case`문으로 `prop`이 `style`이라는게 확실하니깐 `props[prop]` 보다는 `props.style`이렇게 사용해도 가독성을 높일 수 있을 것 같아요!

`($elem.style as any)[key] = props[prop][key]` 저는 이런 식으로 스타일 객체를 덮어씌워 줄 생각은 못했던 것 같네요👍👍👍

Reply:
방금 저도 해보려고 하니깐 `$elem.style`이 readonly 타입이라서 직접적으로 재할당은 허용하지 않는 것 같아요!
아마 영재님은 any로 타입 단언하셔서 들어가는 것 같아요
```ts
interface ElementCSSInlineStyle {
    readonly attributeStyleMap: StylePropertyMap;
    readonly style: CSSStyleDeclaration; // style 객체(readonly)
}
```
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style 여기 MDN 글 읽어 보니깐 `따라서 다른 스타일 값을 변경하지 않고 요소에 특정 스타일을 추가하려면 일반적으로 개체에 개별 속성을 설정하는 것이 좋습니다` 라고 나와있긴 하네용